### ISSUE TITLE
Regular Updates for Members Page

### ISSUE THREAD

**Issue by Frymasterspeck** [codeisland/codeisland.github.io#22]
As we are pulling in new people on a regular basis, we might want to make it a regular function of each meetup to run the members script and push it to the site.

**Comment by goldenmeanie** [comment]
Agree! If we move the site to Digital Ocean we will be able to automate the members updates, but for now it is indeed a manual process.

**Comment by andrewborstein** [comment]
Well, the new implementation on the [asb-theme branch](http://andrewborstein.github.io/code-island.org/) is actually automated already. 

If you visit the [Join page](http://andrewborstein.github.io/code-island.org/join/) and look under "Contributing Members" you will see 18 folks listed. That's the exact number of Code Island Github [team members](https://github.com/orgs/codeisland/people) that have manually changed their visibility to 'public'. This list is implemented via the GitHub API, so going forward, new members need to only do two things in order to be listed: 1. contact Ryan Kelley to become a member of the Code Island organization on Github, and then 2. change their membership from private to public. Those directions are also included on the Join page under that heading. Speaking of which, 19 out of 37 total contributors have not yet changed their membership to public.
